packages/js/customer-effort-score/src/customer-feedback-modal/index.tsx

```diff
@@ -112,7 +112,8 @@ function CustomerFeedbackModal( {
 		if (
```

Comment:
This isn't behavior quite as expected, and it might be simpler to use a condition like this:
```js
		if (
			! Number.isInteger( firstQuestionScore ) ||
			( secondQuestion && ! Number.isInteger( secondQuestionScore ) )
		) {
			setShowNoScoreMessage( true );
			return;
		}
```

Reply:
Thanks for catching this. You are totally right. I missed the initial ! before the conditional. I fixed it in https://github.com/woocommerce/woocommerce/pull/36270/commits/5985838eb886bf592199975fbcfee7410959f300